cms/envs/acceptance.py

```diff
@@ -98,7 +98,7 @@ def seed():
 USE_I18N = True
 
 # Include the lettuce app for acceptance testing, including the 'harvest' django-admin command
-INSTALLED_APPS += ('lettuce.django',)
+INSTALLED_APPS += ('lettuce.django', 'django.contrib.staticfiles')
```

Comment:
Gotcha. Agreed that should happen, and agreed with the subtext that this kind of change would be out of scope right now. Thanks for explaining & providing the links offline!